src/components/Entry/EntryTabs.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react'
+import { Container, Form } from 'react-bootstrap'
+import LogoGrey from '../../assets/logo-grey.png'
+
+let EntryTabs = () => {
+    let [tabShow, setTabShow] = useState("left")
+
+    return (
+        <>
+            <Container className="pt-5">
+                <section className="d-flex flex-column justify-content-center align-items-center">
+                    <img alt="grey logo" src={LogoGrey} />
+                    <section className="tabs-container">
+                        <div className="tabs-container-header">
+                            <div onClick={() => setTabShow("left")} className="tabs" style={tabShow === "left" ? { backgroundColor: "#352C50", color: "#fff" } : { backgroundColor: "#f1f1f1", color: "#000" }} ><h5>New User</h5></div>
```

Comment:
The code in this file is quite wide. I suggest using prettier (or another linter) to format it.